src/status_im/contexts/communities/overview/view.cljs

```diff
@@ -366,14 +367,30 @@
            ;; might have been removed.
            on-first-channel-height-changed}]]))))
 
-(defn- community-card-page-view
+(defn- community-fetching-placeholder
   [id]
+  (let [fetching? (rf/sub [:communities/fetching-community id])]
+    [rn/view
+     {:style               style/fetching-placeholder
+      :accessibility-label (if fetching?
+                             :fetching-community-overview
+                             :failed-to-fetch-community-overview)}
+     [not-implemented/not-implemented
+      [rn/text
+       (if fetching?
+         "Fetching community..."
+         "Failed to fetch community")]]]))
+
+(defn- community-card-page-view
+  [community-id]
   (let [{:keys [id joined name images]
-         :as   community} (rf/sub [:communities/community id])]
-    (when community
-      (when joined
-        (rf/dispatch [:activity-center.notifications/dismiss-community-overview id]))
-      [community-scroll-page id joined name images])))
+         :as   community} (rf/sub [:communities/community community-id])]
+    (if community
+      (do
+        (when joined
+          (rf/dispatch [:activity-center.notifications/dismiss-community-overview id]))
```

Comment:
This call will be made on every Reagent pass of this component. Better not do it @yqrashawn, even if it's acceptable here. If, for any reason, another dev changes the code and cause `community-card-page-view` to render more than once (easy to happen) we will fire multiple events. Here you might want to use a `use-effect` with the dependency on `joined`.

Reply:
I moved it to `:communities/handle-community`